modules/users/client/controllers/admin/user.client.controller.js

```diff
@@ -42,8 +44,10 @@
         $state.go('admin.user', {
           userId: user._id
         });
+        Notification.success('User updated successfully!');
       }, function (errorResponse) {
         vm.error = errorResponse.data.message;
+        Notification.error('User update error!');
```

Comment:
@mleanos I had it set that way on the angular-toastr, looks like i kept it generic on this one. It is good idea to have the error on the toast and add title.

Removing vm.error would fail some tests that are checking it. Didn't find any harm in having it until we decided to use these notifications.

I will update the error message with title and post a screenshot. Will update tests, although frankly I have no idea how the test scripts are setup(never looked other than running them).